core/ide/src/main/java/org/alice/stageide/StageIDE.java

```diff
@@ -235,7 +235,7 @@ private Icon getIconFor(AbstractField field) {
           JavaField javaField = (JavaField) field;
           org.lgna.story.Color color = (org.lgna.story.Color) ReflectionUtilities.get(javaField.getFieldReflectionProxy().getReification(), null);
           Color awtColor = EmployeesOnly.getAwtColor(color);
-          return new ColorIconFactory(awtColor).getIcon(new Dimension(15, 15));
+          return new ColorIconFactory(awtColor).getIconToFit(new Dimension(15, 15));
```

Comment:
We didn't have enough different sizes and couldn't use 16x16. Oy.

Reply:
this actually looks fine as an XS square, so i'm improving all of our lives